### PR TITLE
Mvs 627 get immunization history

### PR DESCRIPTION
**Pull request by nxm9892**
## :floppy_disk: [Code Check]
_Make sure you've checked/done the following before making this PR!_
* Does your code build/run? Yes
* Does your design approach make sense? Yes
* Does your work achieve the intention of the card? Yes
* Does your code have any unintentional side effects -- have you tested a full workflow & checked the logic branches you touched? No
* Did you make a unit test for your code (as appropriate)? Antony wrote a test for it.
* Is your branch relatively up to date with the Development branch (within ~2 commits) Yes
* Have you moved the card to "Needs QA" in Jira Yes

## :flipper: [JIRA Sprint Card]

https://mass-immunization-system.atlassian.net/browse/MVS-627

## :newspaper: [Work Description]

1. Patient history component displays the patient's immunization history.

## :vertical_traffic_light: [Testing/QA Notes]

1. Follow README.md instructions to setup Docker/broker/POD App
2. Follow directions at the top of file: EndpointsUnitTest.js to run the tests in this file.
3. Verify through patient lookup that Joe Bush 11/11/1999 has immunization history.

### CODE REVIEW

**Review by peterdobbs77 — COMMENTED** [review]
Does the unit test actually test the functionality?

File: broker/testing/patientHistoryTest.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  );
+  let completed = globals.fhirServer.put(
+    "/Immunization/examplecompleted",
+    JSON.parse(globals.examples.ExampleImmunizationCompleted)
+  );
+  await Promise.all([notDone, completed]).catch((error) => {
+    console.log("Patient history setup failed");
+    globals.info(error);
+  });
+}
```
[the remark this answers is not in the document]
Comment (nxm9892) [reply broker/testing/patientHistoryTest.js:20]: It creates a patient with past immunizations for the purpose of bringing up patient history and the patient history component. So yes.

[the remark this answers is not in the document]
Comment (peterdobbs77) [reply broker/testing/patientHistoryTest.js:20]: All I see is PUTing Immunization resources to the FHIR server. I don't see any coverage for retrieving previous Immunization records for a Patient.

**Comment by peterdobbs77** [comment]
Finally got my system to test this properly, but it looks like you'll need to fix the merge conflict

**Comment by nxm9892** [comment]
Okay. I'll be looking at this later tonight. Sorry for the delay.

**Review by peterdobbs77 — APPROVED** [review]
Looks like it should be good

**Pull request closed without merging**